app/code/Transoft/Blog/etc/db_schema.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Why do we need InstallSchema if you used declarative schema?

Reply:
removed install schema script